projects.yml

```diff
@@ -5442,3 +5460,345 @@ truegrit:
   url: https://github.com/KingMob/TrueGrit
   categories: [Retrying Failures]
   platforms: [clj]
+
+ambly:
+  name: Ambly
+  url: https://github.com/mfikes/ambly
+  categories: [REPLs]
+  platforms: [cljs]
+
+pyramid:
+  name: Lacina
+  url: https://github.com/lilactown/pyramid
+  categories: [Graphs, EQL]
+  platforms: [clj, cljs]
+
+asami:
+  name: Asami
+  url: https://github.com/threatgrid/asami
+  categories: [Graphs]
+  platforms: [clj, cljs]
+
+clj_docker_client:
+  name: clj-docker-client
+  url: https://github.com/into-docker/clj-docker-client
+  categories: [Docker Integration]
+  platforms: [clj]
+
+contajners:
+  name: contajners
+  url: https://github.com/lispyclouds/contajners
+  categories: [Docker Integration]
+  platforms: [clj]
+
+clj_statecharts:
+  name: clj-statecharts
+  url: https://github.com/lucywang000/clj-statecharts
+  categories: [State Machines]
+  platforms: [clj, cljs]
+
+statecharts:
+  name: statecharts
+  url: https://github.com/fulcrologic/statecharts
+  categories: [State Machines]
+  platforms: [clj, cljs]
+
+obb:
+  name: obb
+  url: https://github.com/babashka/obb
+  categories: [Scripting]
+  platforms: [cljs]
+
+scittle:
+  name: scittle
+  url: https://github.com/babashka/scittle
+  categories: [ClojureScript Development, Web Frameworks]
+  platforms: [cljs]
+
+uri:
+  name: uri
+  url: https://github.com/lambdaisland/uri
+  categories: [Utilities]
+  platforms: [clj, cljs]
+
+jirazzz:
+  name: jirazzz
+  url: https://github.com/rwstauner/jirazzz
+  categories: [Scripting]
+  platforms: [clj]
+
+bencode:
+  name: bencode
+  url: https://github.com/nrepl/bencode
+  categories: [Encoding/Decoding]
+  platforms: [clj]
+
+babashka_nrepl:
+  name: babashka.nrepl
+  url: https://github.com/babashka/babashka.nrepl
+  categories: [REPLs]
+  platforms: [clj]
+
+coffi:
+  name: coffi
+  url: https://github.com/IGJoshua/coffi
+  categories: [Foreign Function Interface]
+  platforms: [clj]
+
+malli:
+  name: malli
+  url: https://github.com/metosin/malli
+  categories: [Schema]
+  platforms: [clj, cljs]
```

Comment:
`malli` is already listed in Clojure Toolbox, please remove this

Reply:
Good catch.